src/engine.c

```diff
@@ -485,19 +485,28 @@ defaultFontLookup(utf8_int32_t codepoint) {
 
 internal void
 ENGINE_print(ENGINE* engine, char* text, int64_t x, int64_t y, uint32_t c) {
+  int newlines = 0;
   int fontWidth = 8;
   int fontHeight = 8;
+  int spacing = (fontHeight / 4);
   int cursor = 0;
   utf8_int32_t codepoint;
   void* v = utf8codepoint(text, &codepoint);
   size_t len = utf8len(text);
   for (size_t pos = 0; pos < len; pos++) {
+    if (text[pos] == '\n') {
+      newlines++;
```

Comment:
We can just modify the `y` value directly, instead of counting the newlines.

Reply:
Ah yeah that's true